src/unity/python/turicreate/toolkits/object_detector/object_detector.py

```diff
@@ -1072,7 +1072,6 @@ class cannot be returned and are ignored under this format.
         mAP: 43.2%
         """
         if iou_threshold is None: iou_threshold = self.non_maximum_suppression_threshold
-        if confidence_threshold is None: confidence_threshold = 0.25
```

Comment:
With this change, you're still passing a value for `confidence_threshold` below, it just happens to be None. This will still override the default value for this parameter in `_predict_with_options`, right?

Reply:
My bad, you're right. That's how code works, mental error.

Reply:
Added confidence threshold